dd-sdk-android/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Doesn't this already exist in your other PR ?

Reply:
It is bu that one is not merged yet :(, waiting for the PR on the "dd-trace-java" SDK